manual/running.tex

```diff
@@ -45,25 +45,7 @@ \section{Input files}
 The input is one or more XML file(s), documented in section~\ref{chap:input_overview}.
 
 \section{Output files}
-QMCPACK generates multiple files which have the following suffixes:
-\label{sec:outputs}
-\begin{description}
-\item[\texttt{scalar.dat}]{ This file contains information about
-  the simulation such as the energy, variance, MC acceptance ratio, etc..
-  In many situations, this is the main output file you should use to process
-  the results of the simulation. QMCPACK includes a useful tool for easy
-  processing of these files, called \texttt{qmca}. It is located in the
```

Comment:
The reference to qmca was lost.  It should still be mentioned, along with a link the the chapter on Analyzing QMCPACK data (chapter 11) (assuming that chapter will get filled in eventually)